### DISCUSSION TITLE
Bar is blank (nix-darwin)

### DISCUSSION THREAD

**Discussion by mmorley0395** [FelixKratz/SketchyBar#595] [Q&A]
Hi all, 

I installed via Nix-Darwin rather than brew. The bar is blank, despite having the default config and plugins at the right location. Any tips or example configs for nix-darwin users?

**Comment by art1es23** [comment]
> Hi all,
> 
> I installed via Nix-Darwin rather than brew. The bar is blank, despite having the default config and plugins at the right location. Any tips or example configs for nix-darwin users?

Same issue when try to use it with nix-darwin and home-manager

**Comment by neversad-dev** [comment]
I encountered the same issue, and here’s how I resolved it:

I moved my entire SketchyBar configuration—including the sketchybarrc file and all associated shell scripts—into a centralized config directory. Then, I symlinked this directory in my Home Manager configuration with the following line:

```
home.file.".config/sketchybar".source = ./config;
```

**Reply by dengo123** [reply]
i dont use home manager, how do I do this within a regular flake set up?

**Comment by mmorley0395** [comment]
I had a similar fix, but I simply ran the `sketchybar --reload [Optional: <path>]` command with a path to my dotfiles repo (managed with homemanager) in the optional path param. This seems to have persisted with reboots etc, I only set it once.

**Reply by neversad-dev** [reply]
While this solution works, I personally wouldn’t go this route, as it somewhat breaks the single source of truth principle. Just my perspective, though!

**Reply by dengo123** [reply]
totally not working for me

**Comment by mcspud** [comment]
I'm in the same position as you kind sir @dengo123 
This is my first time using sketchybar (new year, new me I guess) and I can't get it to run using a standard `flake.nix` setup.
